ExtLibs/MV04.Camera/CameraHandler.cs

```diff
@@ -87,6 +87,8 @@ public static CameraHandler Instance
             }
         }
 
+        public static readonly int TripChannelNumber = 4;
```

Comment:
A 4-es a MAIN OUT láb, ahova az első (0-ás) RELAY be van kötve, a doCommand-ban az utóbbit kell küldeni.